.appveyor.yml

```diff
@@ -16,19 +23,18 @@ for:
       - image: Visual Studio 2017
   install:
     - choco install -y dotnetcore-sdk --version=3.1.301
-    - choco install -y winpcap
+    - ps: .\scripts\InstallNpcap.ps1
```

Comment:
@chmorgan The only reason I used the Visual Studio 2017 image was simply to test SharpPcap with WinPcap
I think this should remain as is as a backward compatibility test

Reply:
agree, I kept the winpcap tests